meta-cube/recipes-core/lxc/files/lxc-overlayrestore.bak

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -
+
+# /usr/lib64/lxc/lxc/lxc-overlayrestore
```

Comment:
I see from the above copy command why this is here .. but why this is called .bak, is unclear.